VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -1006,8 +1006,8 @@ export abstract class WaitingRoomBaseDirective implements AfterContentChecked {
     }
 
     async publishMediaDeviceStatus() {
+        // TODO: make hearing subscribe to store or make `audioOnly` an input
```

Comment:
is this for the next pr?

Reply:
I forgot about this TODO: will address it in this PR and test it